src/documents/CoreAPI.md

```diff
@@ -0,0 +1,635 @@
+---
+title: Core API
+layout: default
+category: Infusion
+---
+
+# Infusion Core API
+
+This file documents some low-level APIs which are provided by Infusion at the JavaScript level. Note that since
+Infusion's more overarching aim is to enable declarative programming based on JSON structures, by means
+of its [IoC](FrameworkConcepts.md#ioc) system, the documentation in this file doesn't cover very much
+of Infusion's real function. Please consult pages on [Component Grades](#ComponentGrades.md), 
+[Component Configuration Options](#ComponentConfigurationOptions.md), the [Infusion Event System](#InfusionEventSystem.md)
```

Comment:
All of these links should NOT have the hash (#) in them; that makes them links to anchors on the current page, which is wrong.